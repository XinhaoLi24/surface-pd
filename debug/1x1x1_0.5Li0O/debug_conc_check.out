 Original concentration ranges:
   1    1    2
   1    1    2

 Numbers for given cell size:
   2    2    4
   2    2    4

 Number of discrete partitions:     1

 generated list:
   2    2
