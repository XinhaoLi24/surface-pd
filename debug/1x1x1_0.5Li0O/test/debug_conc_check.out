 Original concentration ranges:
   1    1    4
   3    3    4

 Numbers for given cell size:
   2    2    8
   6    6    8

 Number of discrete partitions:     1

 generated list:
   2    6
